src/frontend/src/icp/components/send/IcSendReview.svelte

```diff
@@ -26,4 +27,6 @@
 		invalidAmount(amount);
 </script>
 
-<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={invalid} />
+<SendReview on:icBack on:icSend {source} {amount} {destination} disabled={invalid}>
+	<IcReviewNetwork {networkId} slot="network" />
```

Comment:
Did you check that in case of ck conversions, both source and destination network are still displayed correctly?

Reply:
Yes, all good there, I attached one more screenshot.

Reply:
Thanks!